morganb-hw03.py

```diff
@@ -0,0 +1,128 @@
+#################################################################################
+# Author: Baella Morgan
+# Username:morganb
+#
+# Assignment:Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:Continue practicing fucntions and using the turtle library.
+# Google Doc Link:https://docs.google.com/document/d/1NjbTxTAXg0Cuo1YH1nipYNSgiDWNQ2J3X0UAzkHL87w/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: turtle documentation page, geeksforgeeks.org, and Ruben Quinonez for help with the arch loop and troubleshooting.
+#
+#
+#################################################################################
+
+import turtle
+
+
+
+
+
+
+
+
+
+
+def edoras(turtle):
+    """
+    Example docstring for function_1. function_1 is not a good function name and should be changed.
+    """
+    turtle.left(30)
```

Comment:
I would never name a turtle `turtle`. What happens is the turtle library from `import turtle` gets overwritten by the turtle instance, causing a runtime error. 

Always name it something unique that shouldn't collide with a module or any other object in your code.